chef-utils/lib/chef-utils/dsl/virtualization.rb

```diff
@@ -140,6 +140,26 @@ def vmware_host?(node = __getnode)
         node.dig("virtualization", "system") == "vmware" && node.dig("virtualization", "role") == "host"
       end
 
+      # Determine if the current node is virtualized on VMware Desktop (Fusion/Player/Workstation).
+      #
+      # @param [Chef::Node] node
+      #
+      # @return [Boolean]
+      #
+      def vmware_desktop?(node = __getnode)
```

Comment:
would it  make more sense to make these `vmware_on_desktop?` so that someone didn't think this was a helper to use on the hypervisor host?

Reply:
I think I went with "vmware_desktop" because it's a term in the HashiCorp Vagrant universe, addressing the Fusion/Player/Workstation products.

As far as I am aware, Chef isn't really working on VMware hosts anymore, as GSX server is long gone (2009) and classic ESX which was based on Redhat went away in 2011. Not sure if there are some wacky workarounds though...